packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -272,7 +272,7 @@ export class Copy {
                 /*
                 * If the source is a PDS and no member was specified then abort the copy.
                 */
-                if((sourceDataSetObj.dsorg == "PO" || sourceDataSetObj.dsorg == "POE") && sourceMember == undefined){
+                if((sourceDataSetObj.dsorg == "PO" || sourceDataSetObj.dsorg == "PO-E") && sourceMember == undefined){
```

Comment:
Should we change all of these to `startsWith("PO")` ? 😋 
Otherwise, I think we should be using `===` instead of `==`